Sources/RInAppMessaging/EventMatcher.swift

```diff
@@ -80,11 +80,10 @@ internal class EventMatcher: EventMatcherType {
         let events = matchedEvents.get()[campaign.id, default: []] + persistentEvents
         let allTriggersSatisfied = triggers.allSatisfy { isTriggerMatchingOneOfEvents($0, events: events) }
         
-        if campaign.isTooltip {
-            return allTriggersSatisfied && events.contains(where: { $0.type == .viewAppeared })
-        } else {
+        guard campaign.isTooltip else {
```

Comment:
@ts-robih-robihamanto Does this change improve the code coverage?
Or is it an improvement for readability by using `guard`?

Reply:
@malayli 
Yes, It does improve the code coverage, previously the line after else {} is never executed, even though only empty space. But, Xcode shows it as not a covered line of code.